pkg/email/trb_advice_letter_internal_review_test.go

```diff
@@ -44,14 +44,14 @@ func (s *EmailTestSuite) TestTRBAdviceLetterInternalReviewEmail() {
 
 <span style="font-size:15px; line-height: 18px; color: #71767A">Easy Access to System Information</span>
 
-<p>` + input.TRBLeadName + ` has completed a draft advice letter for ` + input.TRBRequestName + `. This advice letter is now ready for internal review. Please take a moment to look over the draft and make any suggestions for improvement.</p>
+<p>` + input.TRBRequestName + ` has a draft advice letter that is now ready for internal review. Please take a moment to look over the draft and make any suggestions for improvement.</p>
```

Comment:
It looks like the test is failing, looks like something to do with an extra newline either in the email itself or in the test that checks it - probably a quick fix, and seems to be the only failing test!

Reply:
It might also be worth a quick test to ensure that this also works if you don't put a `TRBLeadName` param (ensuring that we don't render the lead name line)